src/main/java/net/minecraftforge/event/brewing/BrewingStandFuelEvent.java

```diff
@@ -0,0 +1,110 @@
+package net.minecraftforge.event.brewing;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.ForgeEventFactory;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+import javax.annotation.Nonnegative;
+
+/**
+ * A {@link BrewingStandFuelEvent} is fired when determining the fuel value for an item stack
+ * in a {@link net.minecraft.tileentity.TileEntityBrewingStand brewing stand}. <br>
+ * <br>
+ * The fuel value is defined in {@link Item#getBrewingStandFuel(ItemStack)}.
+ * <br>
+ * This event is meant to allow mods overriding brewing stand fuel values of items from vanilla
+ * or other mods.<br>
+ * <br>
+ * To set the brewing stand fuel value of your own item, override {@link Item#getBrewingStandFuel(ItemStack)}
+ * instead.<br>
+ * <br>
+ * This event is fired from {@link ForgeEventFactory#getBrewingStandFuel(ItemStack)}.<br>
+ * <br>
+ * This event is {@link Cancelable} to prevent later handlers from changing the value.<br>
+ * <br>
+ * This event does not have a result. {@link HasResult}<br>
+ * <br>
+ * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ **/
+@Cancelable
+public class BrewingStandFuelEvent extends Event
+{
+    private final ItemStack stack;
+    @Nonnegative
+    private final int defaultValue;
+    @Nonnegative
+    private int currentValue;
+
+    /**
+     * The constructor.
+     * <p>
+     * <p>Users: call {@link ForgeEventFactory#getBrewingStandFuel(ItemStack)} to reduce extra code.
+     *
+     * @param stack the item stack to check
+     */
+    public BrewingStandFuelEvent(ItemStack stack)
+    {
```

Comment:
Honestly I dislike the concept of this even being an event. It just should be a value added to the ItemStack. Why does this need to be an event?

Reply:
This is to allow mod overrides. An override to the value of a few vanilla items can be found here: https://github.com/MinecraftForge/MinecraftForge/pull/4610/files#diff-0540941665b69a6d2df3219d34fc2755R31

Reply:
I understand what you're wanting. Im just saying it shouldn't be an event. There are ways to override vanilla values without events IF thats something we want to allow to happen.

Reply:
should i make a list of named evaluation functions to override instead?